apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -476,7 +476,7 @@ impl Service<QueryPlannerRequest> for BridgeQueryPlanner {
                 )
                 .await;
             let duration = start.elapsed().as_secs_f64();
-            tracing::info!(histogram.apollo_router_query_planning_time = duration,);
+            tracing::info!(histogram.apollo.router.query_planning.duration = duration, phase = %"plan");
```

Comment:
do not modify this metric for now, as it is already monitored by users in production